content/docs/tasks/traffic-management/egress-gateway.md

```diff
@@ -51,7 +51,35 @@ If we used the [sleep](https://github.com/istio/istio/tree/{{<branch_name>}}/sam
 
 First let's direct HTTP traffic without TLS origination
 
-1.  Create an egress `Gateway` for _edition.cnn.com_, port 80:
+1.  Create an egress `Gateway` for _edition.cnn.com_, port 80.
+
+    If you have [mutual TLS Authentication](/docs/tasks/security/mutual-tls/) enabled in Istio, you must create the
+    `Gateway` in the following way:
+
+    ```bash
+        cat <<EOF | istioctl create -f -
+        kind: Gateway
```

Comment:
needs an `apiVersion`, I believe

Reply:
@ijsnellf Yep, I added it just now, thanks, good catch.